### PR TITLE
HIP 100: Deploy EU868 region plan to the majority of Africa

### PR DESCRIPTION
**Pull request by waveform06**
Added HIP XX: Deploy EU868 region plan to the majority of Africa

Rendered View: https://github.com/waveform06/HIP/blob/main/00XX-deploy-eu868-region-plan-to-the-majority-of-africa.md

### CODE REVIEW

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+* Category: Technical
+* Original HIP PR:
+* Tracking Issue:
+* Vote Requirements: veIOT
+
+## 
+**Summary**
+
+
+The LoRaWAN Alliance has a country by country bottom up approach to allocation of frequencies and does not push for harmonization of a frequency to a region as a top down approach.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:16]:
```suggestion
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+The Helium Foundation recommends that the community provisionally sets the currently unspecified and EU433 only countries in the Africa region to the EU868 frequency plan to match with what we understand from local regulatory authorities like the Africa Telecoms Union (ATU) and local deployers have defined as the future LoRaWAN regional frequency EU868. The LoRa Alliance does not currently specify regional parameters for these countries, or only has EU433 as an available frequency plan, so this would be an addition that the Helium Foundation is proposing for Helium network specific devices.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:18]:
```suggestion
The Helium Foundation recommends that the community provisionally sets the currently unspecified and EU433 only countries in the Africa region to the EU868 frequency plan to match with what we understand from local regulatory authorities like the Africa Telecoms Union (ATU) and local deployers have defined as the future LoRaWAN regional frequency EU868. The LoRa Alliance does not currently specify regional parameters for these countries, or only has EU433 as an available frequency plan, so this would be an addition that the Helium Foundation is proposing for Helium network specific devices. The LoRa Alliance has a country-by-country, or bottom up, approach to allocation of frequencies.
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+
+The Foundation proposes:
+
+
+
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:23]:
```suggestion
The Foundation proposes:

```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 4]
```diff
@@ -0,0 +1,321 @@
+# **HIP XX: Deploy EU868 region plan to the majority of Africa**
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:1]:
```suggestion
# HIP XX: Deploy EU868 region plan to the majority of Africa
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+
+
+
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:4]:
```suggestion

```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+* Author: Adrian Clint, [https://github.com/waveform06](https://github.com/waveform06) and Helium Foundation [https://github.com/dewi-alliance](https://github.com/dewi-alliance)
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:5]:
```suggestion
* Author: [Adrian Clint](https://github.com/waveform06) and [Helium Foundation](https://github.com/dewi-alliance) 
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 7]
```diff
@@ -0,0 +1,321 @@
+# **HIP XX: Deploy EU868 region plan to the majority of Africa**
+
+
+
+* Author: Adrian Clint, [https://github.com/waveform06](https://github.com/waveform06) and Helium Foundation [https://github.com/dewi-alliance](https://github.com/dewi-alliance) 
+* Start Date: 22 Sep 2003
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:6]:
```suggestion
* Start Date: 2023-09-22
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 8]
```diff
@@ -0,0 +1,321 @@
...
+* Category: Technical
+* Original HIP PR:
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:8]:
```suggestion
* Original HIP PR: [#789](https://github.com/helium/HIP/pull/789)
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 9]
```diff
@@ -0,0 +1,321 @@
...
+* Tracking Issue:
+* Vote Requirements: veIOT
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:10]:
```suggestion
* Vote Requirements: veIOT Holders
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 10]
```diff
@@ -0,0 +1,321 @@
...
+
+## 
+**Summary**
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:13]:
```suggestion
## Summary
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 11]
```diff
@@ -0,0 +1,321 @@
...
+
+The Foundation proposes:
+
+
+
+* All Africa region countries currently set to an “Unknown” LoRaWAN regional frequency parameter are set to ETSI [EN300.220-2] standard. Region code EU868 on the Helium network.
+* All Africa region countries currently primarily set to EU433 as the default LoRaWAN regional frequency parameter are set to EU868 on the Helium network.
+* All Africa region countries with EU433 or an 800MHz based LoRaWAN regional frequency parameter are set to EU868 on the Helium network.
+* We approach and communicate with all Telecoms Regulatory Authorities in these countries and confirm Helium provisionally complies with the EU868 regional parameter recommendation from the Africa Telecoms Union, and ask for confirmation that provisional operation is not disallowed in this provisional configuration.
+* The Helium Foundation presents this change and approach to the LoRaWAN Alliance Regional Parameters Working Group.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:28]:
```suggestion
* The Helium Foundation presents this change and approach to the LoRa Alliance Regional Parameters Working Group.
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 12]
```diff
@@ -0,0 +1,321 @@
...
+
+## 
+**Motivation**
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:31]:
```suggestion
## Motivation
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 13]
```diff
@@ -0,0 +1,321 @@
...
+
+
+Africa as a continent is lagging behind in the regulation of LoRaWAN frequency plans for each country, and way behind attempting to harmonize the frequency plans that are in use by countries that have a defined LoRaWAN frequency plan. The local country telecoms regulatory authorities are not proactive with the LoRaWAN Alliance to get a frequency plan defined as it's not as much a priority for them as say 5G.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:34]:
```suggestion
Africa, as a continent, is lagging behind in the regulation of LoRaWAN frequency plans for each country, and additionally haven't attempted to harmonize frequency plans that are in use by countries that have a defined LoRaWAN frequency plan. The local regulatory authorities have not prioritized frequency plan selection as other efforts in 5G have taken precedence.
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 14]
```diff
@@ -0,0 +1,321 @@
...
+
+Helium is being approached by organizations that want to deploy Helium infrastructure in Africa regional countries that do not have a frequency plan or have one that is legacy and is not recommended for future use - EU433. Unfortunately, the legal LoRaWAN frequency varies from country to country within the continent. Even rapidly growing IT-focused countries are having trouble defining the exact legal spectrum on which to operate, unless supported by other organizations or Unions.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:36]:
```suggestion
The Helium network is being approached by organizations that want to deploy compatible infrastructure in Africa that do not have a frequency plan or have one that is legacy and is not recommended for future use - EU433. Unfortunately, the legal LoRaWAN frequency varies from country to country within the continent. Even rapidly growing IT-focused countries are having trouble defining the exact legal spectrum on which to operate, unless supported by other organizations or Unions.
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 15]
```diff
@@ -0,0 +1,321 @@
...
+
+The near term goal is to establish EU868 as a provisional regulation allowing Helium to deploy EU868 hotspots across the Africa Region. Possibly we can make this into a provisional regulation in the near term, in the future LoRaWAN Alliance's RP2-1.0.5 LoRaWAN Regional Parameters document - allowing Helium to deploy EU868 across Africa.  EU868 is already used in the largest African countries, including South Africa, Nigeria, Egypt and Kenya but most countries in Africa do not have any regulations and have no assigned usable frequency plan.  The ETSI [EN300.220-2] standard 868 band is the most likely eventual winner as a defacto standard for IoT devices in most of the world with no regulations, especially in ITU Region 1 (EMEA).
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:38]:
```suggestion
The near term goal is to establish EU868 as a provisional regulation allowing Helium to deploy EU868 Hotspots across the African continent. By making this provisional change to the Helium network, we are able to initiate the process to formally change the LoRa Alliance's RP2-1.0.5 LoRaWAN Regional Parameters document - allowing Helium to deploy EU868 across Africa.  EU868 is already used in the largest African countries, including South Africa, Nigeria, Egypt and Kenya but most countries in Africa do not have any regulations and have no assigned usable frequency plan.  The ETSI [EN300.220-2] standard 868 band is the most likely eventual winner as a de facto standard for IoT devices in most of the world with no regulations, especially in ITU Region 1 (EMEA). 
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 16]
```diff
@@ -0,0 +1,321 @@
...
+
+The near term goal is to establish EU868 as a provisional regulation allowing Helium to deploy EU868 hotspots across the Africa Region. Possibly we can make this into a provisional regulation in the near term, in the future LoRaWAN Alliance's RP2-1.0.5 LoRaWAN Regional Parameters document - allowing Helium to deploy EU868 across Africa.  EU868 is already used in the largest African countries, including South Africa, Nigeria, Egypt and Kenya but most countries in Africa do not have any regulations and have no assigned usable frequency plan.  The ETSI [EN300.220-2] standard 868 band is the most likely eventual winner as a defacto standard for IoT devices in most of the world with no regulations, especially in ITU Region 1 (EMEA). 
+
+
+![itu1 map](images/itu1.png)
+
+
+Pic 1: The ITU Region 1 is shown in the map above. From ITU Radio Regulations 2020
+
+The trigger countries for this HIP are Malawi, Liberia, The Democratic Republic Congo, The Republic of Congo and Ghana who the Foundation are working with on the ground through the OPEN team to establish local LoRaWAN connectivity on EU868.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:46]:
```suggestion
The trigger countries for this HIP are Malawi, Liberia, The Democratic Republic Congo, The Republic of Congo, and Ghana who the Foundation are working with on the ground through the OPEN team to establish local LoRaWAN connectivity on EU868.
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa [hunk 17]
```diff
@@ -0,0 +1,321 @@
...
+
+This frequency selection proposed here would be subject to future government or regulatory updates, or an update from the LoRa Alliance. Since this is a local initiative, it's assumed that Hotspot owners are going to deploy Hotspots on their own behalf and not on behalf of the Foundation.  So this guidance is based on our understanding of the local rules through conversations with the regulations and the community. Ultimately, Hotspot owners are the ones who are assuming the risk of deployment, assertion, and ongoing operation. If a Hotspot owner has specific questions or concerns, they should seek current local regulatory or legal advice.
+
+The LoRaWAN Alliance has a country by country bottom up approach to allocation of frequencies and does not push for harmonization of a frequency plan to a region as a top down approach. There is no specific LoRaWAN Alliance Africa regional task force dedicated for any approach region wide like there is for Australia-NZ or Latin-America.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa:50]:
```suggestion
The LoRa Alliance has a country by country bottom up approach to allocation of frequencies and does not push for harmonization of a frequency plan to a region as a top down approach. There is no specific LoRa Alliance Africa regional task force dedicated for any approach region wide like there is for Australia-NZ or Latin-America.
```

**Comment by waveform06** [comment]
We will be using the cultural meaning of "continent" which includes islands rather than the geographical meaning of a continuous area of land.

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa.md [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+## Summary
+
+This HIP recommends that the community provisionally sets the currently unspecified and EU433 countries in the Africa region to the EU868 frequency plan. This change would match guidance from local regulatory authorities like the Africa Telecoms Union (ATU) and local deployers. The LoRa Alliance does not currently specify regional parameters for all of these countries, or has EU433 as an only/primary frequency plan.
+
+As a summary, this HIP proposes:
+
+* All Africa region countries with an “Unknown” LoRaWAN regional frequency plan are set to EU868 per ETSI [EN300.220-2].
+* All Africa region countries set to EU433 as the default LoRaWAN regional frequency parameter are set to EU868 on the Helium network.
+* All Africa region countries with EU433 or an 800MHz based LoRaWAN regional frequency parameter are set to EU868 on the Helium network.
+* We communicate with all Telecoms Regulatory Authorities in these countries and confirm Helium provisionally complies with the EU868 regional parameter recommendation from the Africa Telecoms Union.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa.md:19]:
```suggestion
* The network, through the Helium Foundation and other participants, communicate with Telecom Regulatory Authorities in these countries and confirm Helium provisionally complies with the EU868 regional parameter recommendation from the Africa Telecoms Union.
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa.md [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+* We present the approach to the LoRa Alliance Regulations and Regional Parameters Working Groups.
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa.md:20]:
```suggestion
* The network, through the Helium Foundation and other participants, present the approach to the LoRa Alliance Regulations and Regional Parameters Working Group.
```

File: 00XX-deploy-eu868-region-plan-to-the-majority-of-africa.md [hunk 20]
```diff
@@ -0,0 +1,149 @@
...
+
+The immediate objective is to promote EU868 as a provisional standard, enabling Helium deployers to roll out EU868 Hotspots across the African continent. By introducing this provisional amendment to the Helium network, there's an opportunity to influence changes in the LoRa Alliance's LoRaWAN Regional Parameters specifications.  Major African countries like South Africa, Nigeria, Egypt and Kenya already utilize EU868. However, several African countries have yet to establish regulations specific to LoRaWAN. The EU868 plan is gaining traction as a potential global standard for IoT devices, particularly in the International Telecommunication Union (ITU) Region 1 (EMEA). The trigger countries for this HIP are Malawi, Liberia, The Democratic Republic of  Congo, The Republic of Congo, and Ghana which the Foundation are working with on the ground through the One Planet Education Network team to establish local LoRaWAN connectivity on EU868 in these countries.
+
+![itu1 map](images/itu1.png)
```
Comment (abhay) [inline 00XX-deploy-eu868-region-plan-to-the-majority-of-africa.md:30]: recommend moving all of these images but we can do that in a separate PR
